### PR TITLE
delete flaky OrdinalOrSingleRangePartitionSelector test

### PR DESCRIPTION
**Pull request by alangenfeld**
this fails often enough to be disruptive

### CODE REVIEW

**Comment by alangenfeld** [comment]
* **#25049** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25049?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alangenfeld and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx [hunk 1]
```diff
@@ -110,69 +107,4 @@ describe('OrdinalOrSingleRangePartitionSelector', () => {
...
-    expect(getByTestId('selection').textContent).toEqual(
-      '{"selectedKeys":[],"selectedRanges":[{"start":{"key":"2022-01-02","idx":1},"end":{"key":"","idx":-1}}]}',
-    );
-
-    // Click an end partition, verify the value is saved
-    await user.click(getByText('Select an ending partition'));
-    await user.click(getAllByTestId(`menu-item-2022-01-03`)[1]!);
-    expect(getByTestId('selection').textContent).toEqual(
-      '{"selectedKeys":[],"selectedRanges":[{"start":{"key":"2022-01-02","idx":1},"end":{"key":"2022-01-03","idx":2}}]}',
-    );
```
Comment (alangenfeld) [inline js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx:168]: it seems this is the step that fails sporadically

**Review by alangenfeld — COMMENTED** [review]
im sending this full delete out of frustration but I am happy to have reviewers coach me towards a better solution. Here are some failure runs

https://buildkite.com/dagster/dagster-dagster/builds/95651#0192206b-6155-449e-864d-6b1be4cf1533
https://buildkite.com/dagster/dagster-dagster/builds/95147#01920ac7-74ae-4c5c-b73c-d9a428c9a0d0
https://buildkite.com/dagster/dagster-dagster/builds/95651#0192206b-6155-449e-864d-6b1be4cf1533
https://buildkite.com/dagster/dagster-dagster/builds/94201#0191e81b-156f-4491-b4da-578a297daf68
https://buildkite.com/dagster/dagster-dagster/builds/94200#0191e811-497a-491e-8841-6052c7128710

File: js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx [hunk 2]
```diff
@@ -110,69 +107,4 @@ describe('OrdinalOrSingleRangePartitionSelector', () => {
       '{"selectedKeys":["CA","MN"],"selectedRanges":[]}',
     );
   });
-
-  it('should support All, Single or Range entry for time partition dimensions', async () => {
```
Comment (salazarm) [inline js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx:114]: could just do `it.skip` and add a comment to ignore.

**Review by salazarm — APPROVED** [review]

File: js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx (hunk 2 above)
Comment (alangenfeld) [reply js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx:114]: looks like we lint error on disabled test 
if we want it back its always here in git history

**Pull request merged**